src/components/utility/Typography/Typography.module.css

```diff
@@ -1,58 +1,48 @@
-/* defined in App.css:
-@import url('https://fonts.googleapis.com/css2?family=Livvic:ital,wght@0,600;0,700;1,600;1,700&display=swap');
-
-body {
-  font-family: "Livvic", -apple-system, BlinkMacSystemFont, 'Segoe UI',
-    'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
-    'Helvetica Neue', sans-serif;
-  font-weight: 600;
-}
-*/
-
-h1, h2, h3 {
+h1,
+h2,
+h3 {
   font-weight: 700; /* bold */
 }
 
-h1, h1.h1Large {
-  font-size: 64px;
-  line-height: 56px;
+h1 {
+  font-size: 2.5rem;
+  line-height: 2.5rem;
```

Comment:
Line height for the `h1` was designed to be denser than standard; please correct the math so that line height of `h1` is proportional as designed. 

One option: `font-size: 2.5rem; line-height: 0.875;` 
Another: `font-size: 2.5rem; line-height: 2.1875rem;`

Reply:
rem should be calculated based on the html size, so whatever is the px we need to divide it by the html px, default is considered 16px, I have explicitly mentioned it

Reply:
I think I haven't been clear. I mean that the h1 designed with font size 64px is designed with a line height 56px (smaller than the font) which means that setting both font size and line height to 2.5rem is not faithfully reproducing the design.

Reply:
Ahhh, I didn't notice this was the mobile 40px font size when I started discussing it. This makes sense now.